Classes/DataProcessing/FlexFormProcessor.php

```diff
@@ -94,4 +133,28 @@ public function process(ContentObjectRenderer $cObj, array $contentObjectConfigu
 
         return $processedData;
     }
+
+    /**
+     * @param array $data Current data-record
+     * @param array $flexformData
+     * @param array $processorConfiguration
+     * @return array
+     */
+    public function processOverrideFields(array $data, array $flexformData, array $processorConfiguration): array
+    {
+        $recordContentObjectRenderer = GeneralUtility::makeInstance(ContentObjectRenderer::class);
+        $data = array_merge($data, $flexformData);
+        $recordContentObjectRenderer->start($data);
+
+        $typoScriptService = GeneralUtility::makeInstance(TypoScriptService::class);
+        $overrideFields = $typoScriptService->convertTypoScriptArrayToPlainArray($processorConfiguration['overrideFields.']);
+        $jsonCE = $typoScriptService->convertPlainArrayToTypoScriptArray(['fields' => $overrideFields, '_typoScriptNodeValue' => 'JSON']);
```

Comment:
I would go with jsonContentElement

Reply:
The code is a copy of the lines from https://github.com/TYPO3-Headless/headless/blob/c2eacfc656d62e225254eab6b8e7d3c9ed332ebc/Classes/DataProcessing/DatabaseQueryProcessor.php#L136 

Can be changed to `JsonDecoder->decode`. What do you mean by jsonContentElement?